PresentScreenings/Controllers/FilmInfoDialogController.cs

```diff
@@ -228,7 +230,10 @@ void InitiateSummaryFieldText()
             }
             else
             {
-                SetSummaryFieldText(@"Please, hit the URL button to get film information from the web site.", true);
+                var popoverToText = new Dictionary<bool, string> { };
+                popoverToText[true] = $"Sorry, no description found for {_film}";
+                popoverToText[false] = "Please, hit the URL button above to get film information from the web site.";
+                SetSummaryFieldText(popoverToText[BehaveAsPopup], true);
```

Comment:
Please consider if this local dictionary isn't overkill.

Reply:
Not only overkill, but superfluous, because this popover is not started from the Screening Info dialog when no information exists.
Dropped the `dict` and kept the one useful text.